themes/app/src/scss/_base.scss

```diff
@@ -14,11 +14,18 @@ p {
     margin: 0;
 }
 
-main {
-    text-align: center;
-}
-
 .color-bar {
     margin-left: 0%;
     margin-right: 0%;
 }
+
+.page-content__main {
+    img {
```

Comment:
This is where your typography styles will come in to play.

I would advise on removing these styles and creating a new file called `typography.scss` and in there you will take care of all the styling each element from the WSYWIG produces

The `typography` piece can be handled in a separate story